backend/src/services/mural.service.ts

```diff
@@ -3,7 +3,16 @@ import { UpdateOptions } from "sequelize";
 
 export class MuralService {
   public async create(mural: MuralInterface) {
-    const createdMural: Mural = await Mural.create<Mural>(mural);
+    const coordinates = {
```

Comment:
this part is weird maybe, the intent is to have just longitude and latitude passed from client for simplicity, but on the databse side we have to package them into an object as so when we create a mural... anyone know a better way?